application/common/models/Password.php

```diff
@@ -20,10 +20,10 @@ public function rules(): array
                 'created_utc', 'default', 'value' => MySqlDateTime::now(),
             ],
             [
-                'expiration_utc', 'default', 'value' => MySqlDateTime::now(),
```

Comment:
it's just a default value, it will be dealt with appropriately in the behaviors before insert.